go-controller/pkg/util/ovs.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 const (
-	ovsCommandTimeout = 5
+	ovsCommandTimeout = 15
```

Comment:
I think a comment would be good here explaining why the timeout has been increased that much.

Reply:
Sure.